matter_server/server/device_controller.py

```diff
@@ -383,15 +384,24 @@ async def open_commissioning_window(
     @api_command(APICommand.DISCOVER)
     async def discover_commissionable_nodes(
         self,
-    ) -> CommissionableNode | list[CommissionableNode] | None:
+    ) -> CommissionableNodeData | list[CommissionableNodeData] | None:
         """Discover Commissionable Nodes (discovered on BLE or mDNS)."""
         if self.chip_controller is None:
             raise RuntimeError("Device Controller not initialized.")
 
         result = await self._call_sdk(
             self.chip_controller.DiscoverCommissionableNodes,
         )
-        return result
+
+        def convert(cn: CommissionableNode) -> CommissionableNodeData:
+            cnd = CommissionableNodeData()
+            for field in CommissionableNodeData.__dataclass_fields__:
```

Comment:
linter reports:

```
matter_server/server/device_controller.py:398:25: E1101: Class 'CommissionableNode' has no '__dataclass_fields__' member (no-member)
```

I'm not quite sure the issue - could be the linter is confused about `CommissionableNode` from `chip.ChipDeviceCtrl` vs `chip.discovery` - however since it's a subclass, I would expect it to have all the same fields available. Could be that `__dataclass_fields__` field is created in some way that the linter is unable to detect.

Should I just ignore the lint warning here? Or are there any ideas to fix it?

Reply:
Yeah, its because they subclassed with a completely different signature and even with the same name.
Let's ignore for now yeah, I don't have any better options.